### PR TITLE
added templates from 471

### PR DESCRIPTION
**Pull request by thessbsu**
#37
copied the templates that Professor Dit added to our 471 repo into the same location.
based on what I read and placement in the other repo these should then show up as options when creating a new issue

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/4-epic.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+name: 4 Epic
+about: Epic that will be refined into one or more user stories
+title: ''
+labels: epic-story
```
Comment (josephmoore99) [inline .github/ISSUE_TEMPLATE/4-epic.md:5]:
```suggestion
labels: epic
```
I think we just have 'epic' for our label.

**Review by josephmoore99 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/1-task.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+name: 1 Task
+about: Task that should be linked to a user story
+title: ''
+labels: ''
```
Comment (AshlynAdamson) [inline .github/ISSUE_TEMPLATE/1-task.md:5]:
```suggestion
labels: 'task'
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+The pull request should be linked to either:
+ - a task (i.e., use 'Closes #TaskID' or 'Resolves #TaskID')
+ - a bug (i.e., use 'Closes #BugID' or 'Fixes #BugID')
+
+For more details, see: https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword
+-->
+
+Closes #TBDTaskIDorBugID
+
+TBDAdditionalPRDetails
```
Comment (AshlynAdamson) [inline .github/pull_request_template.md:13]: I wonder if there's an assignees field for pull requests, then we could put all of our names on it and have that piece automated as well.

Comment (AshlynAdamson) [reply .github/pull_request_template.md:13]: We discussed this in our weekly scrum, and we may use a different form of communication for PRs

**Review by AshlynAdamson — APPROVED** [review]

**Pull request merged**